tests/test_live_api.py

```diff
@@ -75,15 +75,26 @@ def test_get_itinerary(self):
 
     def test_get_image_url(self):
         url = self.dossier.get_map_url()
-        self.assertIsInstance(url, basestring)
+
+        if sys.version_info.major < 3:
```

Comment:
Hmm, on second thought. I notice it differs all over the place because of the variable being asserted ... Might just be something to live with until everyone is on Python 3 🤔 

Additionally, we can eventually release a `gapipy==2.0.0` version which is Python 3 only and not backwards compatible. Django is doing the same thing with Django `2.0.0`